test/end-to-end/reports/clients_report/clients_report.spec.js

```diff
@@ -0,0 +1,48 @@
+/* global browser, element, by */
+
+const chai = require('chai');
+const helpers = require('../../shared/helpers');
+
+helpers.configure(chai);
+
+const ReportClientsPage = require('./clients_report.page');
+
+// @FIXME skip end to end tests until form validation and components are finalised
+describe('Clients report ::', () => {
+  let Page;
+  const key = 'clients_report';
+
+  const dataset = {
+    start_date : '01/01/2016',
+    end_date : '31/12/2016',
+    clients : ['First Test Debtor Group'],
+    report_name : 'Clients Report Saved by E2E',
+    renderer : 'PDF'
+  };
+
+  before(() => {
+    helpers.navigate(`#!/reports/${key}`);
+    Page = new ReportClientsPage(key);
+    browser.refresh();    
```

Comment:
Our end to end test see,s strange, it needs an investigation. For now we are refreshing the page to let test pass otherwise only the first report test will work.